testsuite/MDAnalysisTests/core/test_topologyattrs.py

```diff
@@ -335,6 +335,11 @@ def test_principal_axes(self, ag):
                       [1.20986911e-02, 9.98951474e-01, -4.41539838e-02],
                       [-9.99925632e-01, 1.21546132e-02, 9.98264877e-04]]))
 
+    def test_principal_axes_handedness(self):
+        universe = mda.Universe(PDB_CHECK_RIGHTHAND_PA)
```

Comment:
To anticipate @orbeckst comments, I think this is better off as a [fixture](https://docs.pytest.org/en/latest/fixture.html#fixtures-as-function-arguments). Something like
```
@pytest.fixture
def universe_pa():
    return mda.Universe(PDB_CHECK_RIGHTHAND_PA)

def test_principal_axes_handedness(self, universe_pa):
    pass
```